web/themes/custom/novel/templates/fields/field--field-hero-categories.html.twig

```diff
@@ -0,0 +1,4 @@
+{% for item in items %}
+    {% set item = item|merge({'attributes': item.attributes.addClass('tag--large')}) %}
```

Comment:
shouldnt this also have the `tag` class? right now you're only setting `tag--large`

Reply:
The tag class is in the `field--field-categories.html.twig`